resources/prometheus/remote-write.yaml

```diff
@@ -33,20 +33,86 @@ writeRelabelConfigs:
       - __name__
     targetLabel: __tmp_keep
 
+  # WHAT: Reports the current amount of samples that have tried to be resent to observatorium
+  - action: replace
+    regex: kube_pod_info$
+    replacement: 'true'
+    sourceLabels:
+      - __name__
+    targetLabel: __tmp_keep
+
+  # WHAT: Reports the current amount of samples that have tried to be resent to observatorium
+  - action: replace
+    regex: container_memory_working_set_bytes$
+    replacement: 'true'
+    sourceLabels:
+      - __name__
+    targetLabel: __tmp_keep
+
+  # WHAT: Reports the current amount of samples that have tried to be resent to observatorium
+  - action: replace
+    regex: kube_pod_container_resource_limits$
+    replacement: 'true'
+    sourceLabels:
+      - __name__
+    targetLabel: __tmp_keep
+
+  # WHAT: Reports the current amount of samples that have tried to be resent to observatorium
+  - action: replace
+    regex: kube_pod_container_resource_requests$
```

Comment:
You can group all `container_network_.*` metrics in one expression

Reply:
Okay @lgarciaaco i have done this